clean.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
```

Comment:
After reading the script it looks like the default is nothing. We should actually fix that. My persona preference would be to default to "-b" as that is what I generally do when I iterate.